web/templates/layout/app.html.eex

```diff
@@ -21,11 +21,19 @@
             <a href="/projects" class="dtc tc ttu">Projects</a>
             <a href="/who" class="dtc tc ttu">Who We Are</a>
             <a href="/faqs" class="dtc tc ttu">FAQS</a>
```

Comment:
Do you think it may be better to use [`link`](https://hexdocs.pm/phoenix_html/Phoenix.HTML.Link.html) provided by phoenix instead of `a` tags?